src/Host.cpp

```diff
@@ -45,6 +45,8 @@
 #include <memory>
 #include "post_guard.h"
 
+const int Host::mLuaSendPasswordTimeout = 120000; // milli-seconds!
```

Comment:
Try the [""min](https://en.cppreference.com/w/cpp/chrono/operator%22%22min) operator:

```suggestion
const int Host::mLuaSendPasswordTimeout = 2min;
```

Reply:
I tried it but it *Ｄｏｅｓｎ＇ｔ  Ｗｏｒｋ  Ｆｏｒ  Ｍｅ！*
![image](https://user-images.githubusercontent.com/6163092/80609653-43df6a80-8a30-11ea-8c1c-e8bf3c37d146.png)
:frowning_face:

Reply:
Can you find out why? It's used other parts of the code.

Reply:
You might need to have a "using" statement!